packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -3,7 +3,7 @@
  * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
  */
 import {waitForValue, WebexPlugin} from '@webex/webex-core';
-import {oneFlight, tap} from '@webex/common';
```

Comment:
No longer required. Not sure why `tap` was used instead of the more straight forward _older_ APIs.